### DISCUSSION TITLE
Add a milestone to a Gitlab PR

### DISCUSSION THREAD

**Discussion by Audie80** [renovatebot/renovate#30212] [Suggest an Idea]
### Tell us more.

The documentation explains how to add a milestone to a Github PR : https://docs.renovatebot.com/configuration-options/#milestone
I think Renovate should have a configuration to add a milestone to a Gitlab PR.
The difference is on Gitlab, milestones don't have "number". But they have "id" : https://docs.gitlab.com/ee/api/milestones.html#get-single-milestone
Thank you !

**Comment by rarkins** [comment]
Created issue: https://github.com/renovatebot/renovate/issues/30281
